.circleci/config.yml

```diff
@@ -648,6 +676,13 @@ workflows:
       - acceptance-eks-1-18
       - acceptance-aks-1-19
       - acceptance-kind-1-21
+      - cleanup-azure-resources:
+          requires:
+            - acceptance-aks-1-19
+            - acceptance-openshift
```

Comment:
I think you need to run the cleanup as a step as part of the actual `acceptance-aks-1-19` or `acceptance-gke-1-17` jobs because as written these jobs will only run if the tests succeed but we want the cleanup to always run. It doesn't look like there's a way to say "run this after this even if it failed" in the workflows config.

Unfortunately we would then still run into the issue of the destroy job being stopped during its run.

Reply:
https://discuss.circleci.com/t/how-to-always-run-a-sequential-final-job-e-g-slack-notification/35362

Reply:
Yeah I was thinking about that too, but unfortunately, since openshift tests are running in parallel, I can't make it a step in the job because then one parallel run could delete resources from another run. Moreover, this could happen for other test jobs if we make them parallel too. So I think this is the "best effort" type of job that will run only if tests succeed. Another option would be to make it a separate workflow, but then we'd have to make sure to time it correctly so that it doesn't overlap with existing tests running.

Reply:
What if it runs *before* the jobs?

Reply:
I think that will still cause race conditions, because you'd have two processes pulling up the same list of resources and trying to delete them simultaneously. I could try though and see if that works.

Reply:
Sorry that I've misunderstood you. Updated to run the cleanup jobs before the test jobs!